docs/src/main/paradox/scala/http/common/xml-support.md

```diff
@@ -4,23 +4,62 @@ Akka HTTP's @ref[marshalling](marshalling.md) and @ref[unmarshalling](unmarshall
 infrastructure makes it rather easy to seamlessly support specific wire representations of your data objects, like JSON,
 XML or even binary encodings.
 
-For XML Akka HTTP currently provides support for [Scala XML](https://github.com/scala/scala-xml) right out of the box through it's
+@@@ div { .group-java }
+
+Akka HTTP does not currently provide a Java API for XML support. If you need to
+handle XML, one possible option is to use [Jackson].
+
+@@ snip [#jackson-xml-support] (../../../../../test/java/docs/http/javadsl/JacksonXmlSupport.java) { #jackson-xml-support }
```

Comment:
There is a ticket for adding a Java API for supporting XML (https://github.com/akka/akka-http/issues/191), however, rather than having an empty page I added a small example.